routing/redirect_trailing_slash.rst

```diff
@@ -27,7 +27,9 @@ new URL with a 301 response status code::
 
             $url = str_replace($pathInfo, rtrim($pathInfo, ' /'), $requestUri);
```

Comment:
Not related, but we can take benefit to fix it: why a space on `rtrim` second argument?

Reply:
Because you want to remove spaces too

Reply:
Why? I tried without, works well.

Reply:
I mean what happens if the url is `http://my.website/someroute/    `? That works too?

Reply:
@greg0ire Yes.

Reply:
But I did it on Chrome, maybe the browser remove it itself? :thinking:

Reply:
Maybe, because I can't see how that would work